src/algorithms/changebonds/svdcut.jl

```diff
@@ -1,12 +1,14 @@
 """
-    struct SvdCut <: Algorithm end
+$(TYPEDEF)
 
 An algorithm that uses truncated SVD to change the bond dimension of a ψ.
 
-# Fields
-- `trscheme::TruncationScheme = notrunc()` : The truncation scheme to use.
+## Fields
+
+$(TYPEDFIELDS)
 """
 @kwdef struct SvdCut <: Algorithm
+    "algorithm used for truncation of the gauge tensors"
```

Comment:
```suggestion
    "algorithm used for [truncation][@extref TensorKit.tsvd] of the gauge tensors"
```

Not really relevant to the core of this PR, but a very frequent complaint is that it is not clear how to find out which truncation algorithms exist, so I think it would be good to systematically link to [`TensorKit.tsvd`](https://jutho.github.io/TensorKit.jl/stable/lib/tensors/#TensorKit.tsvd) since this is the only place where they are actually listed.